x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/playground/playground.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback } from 'react';
+
+import { useValues } from 'kea';
+
+import { i18n } from '@kbn/i18n';
+
+import { KibanaLogic } from '../../../shared/kibana';
+import { NEW_INDEX_PATH } from '../../routes';
+import { EnterpriseSearchContentPageTemplate } from '../layout/page_template';
+
+export const Playground: React.FC = () => {
+  const { searchPlayground, navigateToUrl } = useValues(KibanaLogic);
+  const handleNavigateToIndex = useCallback(() => navigateToUrl(NEW_INDEX_PATH), [navigateToUrl]);
+
+  return (
+    <searchPlayground.PlaygroundProvider navigateToIndexPage={handleNavigateToIndex}>
```

Comment:
@TattdCodeMonkey is going to provide more detail on this, but we should probably use a plugin `locator` here instead of a provider for a single route:

https://docs.elastic.dev/kibana-dev-docs/tutorials/share#locators